src/api/tag/tests.js

```diff
@@ -211,4 +210,17 @@ describe('Tags API', () => {
         expect(body).to.have.length(1);
         expect(body[0]._id).to.equal(articleTwoTags);
       }));
+
+  it('should return articles by topic', () =>
+    request
+      .get('/api/topics/articles/locations')
+      .expect(HttpStatus.OK)
+      .expect(({ body: { articles, articlesByTag, tags, topic } }) => {
+        expect(topic.slug).to.equal('locations');
+        expect(articles).to.have.length(2);
+        expect(tags).to.have.length(1);
+        expect(tags[0].slug).to.equal('miensk');
+        expect(Object.keys(articlesByTag)).to.eql(['miensk']);
```

Comment:
`eql` - is it a new shortcut?

Reply:
https://www.chaijs.com/api/bdd/#method_eql